scheduling-api-graphql/scheduling-api-graphql-common/src/main/java/org/ow2/proactive/scheduling/api/graphql/common/InputFields.java

```diff
@@ -37,8 +37,11 @@
 
     AFTER,
     BEFORE,
+    LOWER_THAN,
```

Comment:
all items in this class is ordered in alphabe, please keep the order. And please add this comments in the code, so that anyone will be aware about this.